keith-vscode/src/pasta/stpa-interaction.ts

```diff
@@ -84,6 +83,4 @@ async function importStpaLTL(currentUri: vscode.Uri): Promise<void> {
  * @param description The description for the annotation,
  * @returns the SCChart LTL annotation for the given arguments.
  */
-const ltlAnnotation = (formula: string, description: string): string => {
-    return "@LTL \"" + formula + "\", \"" + description + "\" \n";
-};
+const ltlAnnotation = (formula: string, description: string): string => `@LTL "${formula}", "${description}" \n`
```

Comment:
Why is this better than the previous version? I think this is now harder to read, what is your opinion?

Reply:
removing all the escaped \" characters is definitely better imo, whether using string concatenation or writing with `${}` could be debated, but I think this is fine. Also what was recommended by Prettier defaults.